pycbc/filter/matchedfilter_cpu.pyx

```diff
@@ -31,6 +31,8 @@ ctypedef fused COMPLEXTYPE:
     float complex
     double complex
 
+@cython.boundscheck(False)
+@cython.wraparound(False)
 def _batch_correlate(numpy.ndarray [long, ndim=1] x,
                      numpy.ndarray [float complex, ndim=1] y,
```

Comment:
This should definitely be COMPLEXTYPE (same as others below)

Reply:
@spxiwh Be careful a bit. This model is taking in a array of pointers. I don't think I tried the templating because it really is assuming particular types to work. I guess one could use the template and then assume all types match the strain data (y).